### PR TITLE
add interview page with collections

### PR DESCRIPTION
**Pull request by meltedspork**
interview page

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -42,6 +42,11 @@ social:
 ########
 # more settings
 
+collections:
+  people:
```
Comment (kratsg) [inline _config.yml:46]: do we want to call it `people`?

File: _includes/component_grid_core.html [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<div class="grid-core" style="background-image: url(img/{{ include.data.id }}.png);">
+<div class="grid-core" style="background-image: url(../img/{{ include.data.id }}.png);">
```
Comment (kratsg) [inline _includes/component_grid_core.html]: just start with a forward slash to be relative to root.

File: _includes/head.html [hunk 3]
```diff
@@ -9,11 +9,11 @@
 
   <link rel="canonical" href="{{ site.url }}" />
 
-  <link rel="shortcut icon" type="image/png" href="img/favicon-32x32.png"/>
+  <link rel="shortcut icon" type="image/png" href="{{ site.baseurl }}/img/favicon-32x32.png"/>
 
   <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
   <link rel="stylesheet" type="text/css" href="{{ site.baseurl }}/css/style.css" >
-  <link rel="stylesheet" type="text/css" href="/css/font-awesome.min.css">
+  <link rel="stylesheet" type="text/css" href="{{ site.baseurl }}/css/font-awesome.min.css">
```
Comment (kratsg) [inline _includes/head.html]: Why was this changed? It was fine before?

**Review by kratsg — CHANGES_REQUESTED** [review]

File: _config.yml (hunk 1 above)
Comment (meltedspork) [reply _config.yml:46]: I have no preference here. I only used `people` based on our last conversation about using collections.

File: _includes/head.html (hunk 3 above)
Comment (meltedspork) [reply _includes/head.html]: We should make it consist here either we use `{{ site.basurl }}` or forward stash to be relative to root. We should avoid mixing both and ends in confusion.

File: _includes/component_grid_core.html (hunk 2 above)
Comment (meltedspork) [reply _includes/component_grid_core.html]: You are right. this does bring good question. we should decide on one way doing the path and be consistance with those.

Comment (kratsg) [reply _includes/component_grid_core.html]: I agree, I think using the forward slash is the best way forward.

File: _includes/head.html (hunk 3 above)
Comment (kratsg) [reply _includes/head.html]: I agree, I think using the forward slash is the best way forward.

**Review by kratsg — APPROVED** [review]
